content/doc/developer/architecture/extensions.adoc

```diff
@@ -2,4 +2,10 @@
 title: Extensions
 layout: developersection
 wip: true
----
\ No newline at end of file
+---
+
+Jenkins defines extension points, which are interfaces or abstract classes that model an aspect of its behavior. Those interfaces define contracts of what need to be implemented, and Jenkins allows plugins to contribute those implementations. In general, all you need to do to register an implementation is to mark it with `@Extension`.
+
+https://www.jenkins.io/doc/developer/extensions/[The extensions index]  is a useful list of the extension points used in Jenkins and Jenkins plugins.
```

Comment:
```suggestion
link:https://www.jenkins.io/doc/developer/extensions/[The extensions index] is a useful list of the extension points used in Jenkins and Jenkins plugins.
```
adding in the `link:` macro to make sure the link is directed properly and a small spacing update